.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
+.vscode/
```

Comment:
These two (and likewise any other OS-specific or IDE-specific) are specific to individuals, and they should be excluded in the individual's global gitignore, instead of in each and every project.